assets/activation-scripts/activate.d/zdotdir/README.md

```diff
@@ -5,7 +5,7 @@ be in a position to configure the environment _last_, after all system and
 user-specific configuration "rc" files have been processed, simply to prevent
 these scripts from perturbing the flox environment.
 
-Unlike `bash`, `zsh` does not support `--rcfile`, `--norc` or `--no-profile`
+Unlike `bash`, `zsh` does not support `--rcfile`, `--norc` or `--noprofile`
```

Comment:
if zsh doesnt support these options then why are we calling zsh with `-o NO_GLOBAL_RCS -o NO_RCS` above? `--no-globalrcs` is mentioned below but this sentence is still giving me mixed messages.

Reply:
This was only fixing a typo in the pre-existing comment; this change isn't functionally related to the PR. But I'm not sure what you mean here - the comment is saying that `zsh` doesn't have the same options supported by `bash`, so we'll need to use other options that work with `zsh`.

Reply:
i think this might just me messing up which file this is in.
but basically my question was how `-o NO_RCS` is different from `--norc` in bash.
I.e. we describe that it needs all this other setup while also setting these options which are not described but look like they are the equivalent of the respective bash options

Reply:
I'll go ahead and resolve this to get it merged, but happy to entertain any rewrites of the comment.